From a81ec5cf6c590223b9c948caa6de82d9a8d1fcf4 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Wed, 18 Apr 2012 19:22:26 +0000
Subject: [PATCH 086/462] Mantis 0022440: Reduce the number of compiler
 warnings added compiler flag -wd4290 to suppress the warning "A function was
 declared using exception specification."

git-svn-id: svn://localhost/SelfServiceBuild/trunk@115 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.common.mak | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SelfServiceBuild/Build/cpp/makefile.common.mak b/SelfServiceBuild/Build/cpp/makefile.common.mak
index cdcb17e..28760c0 100644
--- a/SelfServiceBuild/Build/cpp/makefile.common.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.common.mak
@@ -213,9 +213,9 @@ STDDEFINES       = -DWIN32 -DWINNT $(TAO_DEFINES) -DMSYSTHREAD_USE_SEH
 
 !if "$(VC_COMPILER_VERSION)" == "VC10"
 !if "$(PLATFORM_DEBUG)" == "TRUE"
-STDCFLAGS        = -nologo -c -MDd -W3 -GR -EHsc -Zi -Od $(STDDEFINES) -I$(_INC) -I$(_GEN)
+STDCFLAGS        = -nologo -c -MDd -W3 -GR -EHsc -Zi -Od -wd4290 $(STDDEFINES) -I$(_INC) -I$(_GEN)
 !else
-STDCFLAGS        = -nologo -c -MD -W3 -GR -EHsc -Zi -Od $(STDDEFINES) -I$(_INC) -I$(_GEN)
+STDCFLAGS        = -nologo -c -MD  -W3 -GR -EHsc -Zi -Od -wd4290 $(STDDEFINES) -I$(_INC) -I$(_GEN)
 !endif
 STDDEFINES       = $(STDDEFINES) $(BOOST_DEFINES) -D_CRT_SECURE_NO_WARNINGS -D_CRT_NON_CONFORMING_SWPRINTFS -D_COMPILER_VERSION_VC10
 # to decide optimization level 
-- 
2.41.0.windows.1

